tech/languages/csharp/mono-installation.md

```diff
@@ -1,24 +1,44 @@
 ---
-title: Mono
+title: C#
 subsection: csharp
 section: tech-languages
-order: 1
+order: 4
 version: 7.8.4
-description: "C# provided by Mono: a cross-platform, open source, .NET development framework"
+description: "C# - .NET Core and Mono"
```

Comment:
That is valid description of these pages and their contents, there is not much else to mention that would actually be useful... I don't think that the original description was more descriptive, to the contrary, it said even less.